Document environment variable globbing (#268)
iterative/terraform-provider-iterative: docs/resources/task.md

- `image` - (Optional) [Machine image](#machine-images) to run the task with.
- `parallelism` - (Optional) Number of machines to be launched in parallel.
- `directory` - (Optional) Local directory to synchronize.

> - `environment` - (Optional) Map of environment variable names and values for the task script. Empty string values are replaced with local environment values. When using empty string values and a [glob](<https://en.wikipedia.org/wiki/Glob_(programming)>) name, all the matching variables will be imported.

Reviewer: ```suggestion
- `environment` - (Optional) Map of environment variable names and values for the task script. Empty string values are replaced with local environment values. Empty values may also be combined with a [glob](<https://en.wikipedia.org/wiki/Glob_(programming)>) name to import all matching variables.
```
